sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/fullfidelity/AutoCheckpointer.java

```diff
@@ -51,7 +52,13 @@ public void close(ChangeFeedObserverContext context, ChangeFeedObserverCloseReas
 
     @Override
     public Mono<Void> processChanges(ChangeFeedObserverContext context, List<JsonNode> docs) {
-        return this.observer.processChanges(context, docs)
+        throw new UnsupportedOperationException("processChangesV1() should be called instead for Full Fidelity");
+    }
+
+    @Override
+    public Mono<Void> processChangesV1(ChangeFeedObserverContext context, List<ChangeFeedProcessorItem> docs) {
```

Comment:
if V1 is bind to wire format,  should the namespace/package change from full fidelity -> V1